dev/sbt-plugin/src/main/scala/com/lightbend/lagom/sbt/InternalConfigs.scala

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package com.lightbend.lagom.sbt
+
+import sbt._
+
+object InternalConfigs {
+  private[sbt] val devModeConfig = config("lagom-dev").hide
+  private[sbt] val cassandraDevModeConfig = config("lagom-dev-cassandra").hide
```

Comment:
@jroper I just pushed a new commit. Have a look and let me know if I can squash. Btw, I'll now extend the `distribution` sbt-test to verify that the dev-mode modules are indeed not bundled when building a distribution.